website/docs/en/guide/features/layer.mdx

```diff
@@ -0,0 +1,117 @@
+# Layer
+
+Layer is a powerful feature provided by Rspack that allows you to categorize and manage modules. By assigning different layers to modules, you can perform differentiated processing for different types of modules, such as:
+
+- Using different compilation targets based on different layers
+- Splitting modules from different layers into different output directories
+- Applying different optimization strategies for code in different environments (such as server-side and client-side)
+
+## Ways to specify layers
+
+You can specify layers for modules in the following ways:
+
+### 1. Specify via entry
+
+Directly specify layers for different entry points in the entry configuration:
```

Comment:
Could you make this part more detailed? Does the layer affect only the entry module, or all the modules imported by the entry module as well?